tests/testthat/_snaps/R4.2/selections.md

```diff
@@ -3,7 +3,7 @@
     Code
       recipes_eval_select(quos = quos(I(beds:sqft)), data = Sacramento, info = info_sac)
     Condition
-      Error in `recipes_eval_select()`:
+      Error:
```

Comment:
This one is worse because we are testing the internal function by itself. This is fine as it will also be used within a function